### DISCUSSION TITLE
Saving HTML as UTF-8

### DISCUSSION THREAD

**Discussion by chezjim** [brackets-cont/brackets#286] [Q&A]
WS3.org informs me: "It will not be sufficient to just change the declarations inside your pages to say that the page is encoded in UTF-8. You must ensure that your data is actually encoded, ie. saved, in UTF-8.

If you are working with hand-edited files then you should use the options of your editor to save the file in UTF-8 rather than the encoding you were using. If you are building files from scripts and databases, you should ensure that the data is converted as necessary and that the correct parameters are set in your scripting environment."

Only, I see no way to save pages explicitly as UTF-8 in Brackets. Does it do that automatically?

(My declaration of UTF-8 isn't working for non-English characters.)

**Comment by abose** [comment]
By default, brackets save files as UTF-8 . Non-English characters not showing may indicate that yo do not have the necessary font installed in your system. 

Try if the issue happens in https://phcode.dev

**Reply by chezjim** [reply]
Thanks. This was happening on-line and the same characters appeared find in other code.
Meanwhile I discovered Brackets is not being updated. So I moved to Notepad++ and resolved the problem right off.

Thanks for your help.
